### ISSUE TITLE
TST: CSSWarnings

### ISSUE THREAD

**Issue by jreback** [pandas-dev/pandas#16082]
```
(pandas) bash-3.2$ pytest pandas/tests/io/formats/test_css.py  -s
=========================================================================================== test session starts ===========================================================================================
platform darwin -- Python 3.6.1, pytest-3.0.7, py-1.4.32, pluggy-0.4.0
rootdir: /Users/jreback/pandas, inifile: setup.cfg
plugins: cov-2.3.1, xdist-1.15.0
collected 92 items 

pandas/tests/io/formats/test_css.py ....xx/Users/jreback/pandas/pandas/io/formats/css.py:248: CSSWarning: Ill-formatted attribute: expected a colon in "b=c')"
  'in %r' % decl, CSSWarning)
x/Users/jreback/pandas/pandas/io/formats/css.py:248: CSSWarning: Ill-formatted attribute: expected a colon in 'hello-world'
  'in %r' % decl, CSSWarning)
./Users/jreback/pandas/pandas/io/formats/css.py:248: CSSWarning: Ill-formatted attribute: expected a colon in ' hello-world'
  'in %r' % decl, CSSWarning)
./Users/jreback/pandas/pandas/io/formats/css.py:248: CSSWarning: Ill-formatted attribute: expected a colon in ' hello-world'
  'in %r' % decl, CSSWarning)
.xxxx/Users/jreback/pandas/pandas/io/formats/css.py:155: CSSWarning: Unhandled size: 'blah'
  warnings.warn('Unhandled size: %r' % in_val, CSSWarning)
./Users/jreback/pandas/pandas/io/formats/css.py:155: CSSWarning: Unhandled size: '1a2b'
  warnings.warn('Unhandled size: %r' % in_val, CSSWarning)
./Users/jreback/pandas/pandas/io/formats/css.py:155: CSSWarning: Unhandled size: '1e5pt'
  warnings.warn('Unhandled size: %r' % in_val, CSSWarning)
./Users/jreback/pandas/pandas/io/formats/css.py:155: CSSWarning: Unhandled size: '1+6pt'
  warnings.warn('Unhandled size: %r' % in_val, CSSWarning)
./Users/jreback/pandas/pandas/io/formats/css.py:155: CSSWarning: Unhandled size: '1unknownunit'
  warnings.warn('Unhandled size: %r' % in_val, CSSWarning)
./Users/jreback/pandas/pandas/io/formats/css.py:155: CSSWarning: Unhandled size: '10'
  warnings.warn('Unhandled size: %r' % in_val, CSSWarning)
./Users/jreback/pandas/pandas/io/formats/css.py:155: CSSWarning: Unhandled size: '10 pt'
  warnings.warn('Unhandled size: %r' % in_val, CSSWarning)
./Users/jreback/pandas/pandas/io/formats/css.py:219: CSSWarning: Could not expand "margin: 1pt 1pt 1pt 1pt 1pt"
  CSSWarning)
./Users/jreback/pandas/pandas/io/formats/css.py:219: CSSWarning: Could not expand "padding: 1pt 1pt 1pt 1pt 1pt"
  CSSWarning)
./Users/jreback/pandas/pandas/io/formats/css.py:219: CSSWarning: Could not expand "border-width: 1pt 1pt 1pt 1pt 1pt"
  CSSWarning)
./Users/jreback/pandas/pandas/io/formats/css.py:219: CSSWarning: Could not expand "border-color: 1pt 1pt 1pt 1pt 1pt"
  CSSWarning)
./Users/jreback/pandas/pandas/io/formats/css.py:219: CSSWarning: Could not expand "border-style: 1pt 1pt 1pt 1pt 1pt"
  CSSWarning)
.xxxxxxx...........................................................

================================================================================== 78 passed, 14 xfailed in 0.32 seconds ==================================================================================
```

**Comment by jreback** [comment]
cc @jnothman 

are these normal? should we suppress them

**Comment by jreback** [comment]
closed by #16094

**Comment by jnothman** [comment]
Sorry, I'd presumed the pytest.warns context would silence them too. Thanks for fixing.
